server/WebApiTemplate/Controllers/UsersController.cs

```diff
@@ -0,0 +1,88 @@
+﻿using EmeraldChameleonChat.Services.Model.DTO;
+using EmeraldChameleonChat.Services.Model.DTO.Users;
+using EmeraldChameleonChat.Services.Services.Users;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System.Threading;
+
+namespace EmeraldChameleonChat.Controllers
+{
+    [Route("api/[controller]/[action]")]
```

Comment:
I am of the belief that using the variables for the route template is a bad practice. Our internal class names should not dictate a breaking change to the route for the consumer if it were to be updated in the future, you know? Thoughts?

Reply:
I agree.